subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1156,6 +1162,32 @@ static void le_read_buffer_size(struct net_buf *buf, struct net_buf **evt)
 	rp->le_max_num = CONFIG_BT_CTLR_TX_BUFFERS;
 }
 
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
```

Comment:
@Thalley Should this be as below? Does host built for only Synchronised Receiver use LE Read Buffer Size v2 command?

```suggestion
#if defined(CONFIG_BT_CTLR_ISO)
```

Reply:
ok